.github/workflows/conda-package.yml

```diff
@@ -259,16 +259,16 @@ jobs:
           python -c "import dpnp, dpctl; dpctl.lsplatform()"
           python -c "import dpnp; print(dpnp.__version__)"
 
-      - name: Set tests_path for installed package
-        run: |
-          tests_path=$(python -c "import site; import os; print(os.path.join(site.getsitepackages()[0], 'dpnp', 'tests'))")
-          echo "tests_path=$tests_path" >> $GITHUB_ENV
+      - name: Create test temp dir
```

Comment:
The issue has been already fixed in pytest. Do we really need to have the w/a still?